templates/students/update_profile.html

```diff
@@ -149,4 +152,33 @@ <h2>
         }, false)
     }())
 </script>
+
+<!-- Script for Image Validation and auto-submitting the form-->
+<script type="text/javascript">
+    function imageValidation() {
+        var fileInput = document.getElementById('profileImage');
+        var filePath = fileInput.value;
+        var allowedExtensions = /(\.jpg|\.jpeg|\.png|\.gif)$/i;
+        if (!allowedExtensions.exec(filePath)) {
+            alert('Please upload file having extensions .jpeg, .jpg, .png or .gif only.');
+            fileInput.value = '';
+        }
+        else{
+            document.getElementById("submit").click();
```

Comment:
Instead of clicking in the submit button, submit the form directly. Use `$("#<form-id>").submit()`.

Reply:
Yes, I did that. It was not working IDK why............. I tried many times and invested a lot of time on it then I came up with this workaround.

Reply:
That's weird. It should work. Are you naming the `id` of form correctly?

Reply:
@garg3133 yes, it should absolutely work and I can't understand why it's behaving like this. I am using the correct id too. I was busy solving this yesterday night till 3 a.m. but still not working so I had to use this workaround.